library/pk_internal.h

```diff
@@ -33,6 +33,17 @@
 #include "psa/crypto.h"
 #endif
 
+#if defined(MBEDTLS_PSA_CRYPTO_C)
+#include "mbedtls/psa_util.h"
+#define PSA_PK_TO_MBEDTLS_ERR(status) psa_pk_status_to_mbedtls(status)
```

Comment:
Note to self: depending on the order of merges, either this PR or https://github.com/Mbed-TLS/mbedtls/pull/7665 will have to be updated.